$NetBSD$
BSD systems don't have alloca.h but have alloca in stdlib.h.
http://sourceforge.net/p/qore/code/6300/
--- configure.ac.orig	2009-09-21 11:55:18.000000000 +0000
+++ configure.ac
@@ -579,6 +579,7 @@ AC_CHECK_MEMBERS([struct stat.st_rdev])
 AC_HEADER_TIME
 AC_STRUCT_TM
 AC_C_VOLATILE
+AC_CHECK_HEADERS([alloca.h])
 
 # turn on optimizations if we know how
 if test "$enable_optimization" = "yes"; then
